src/lib/import-export/import/importers/Importer.ts

```diff
@@ -0,0 +1,67 @@
+import fsPromises from 'fs/promises';
+import path from 'path';
+import { BackupContents } from '../types';
+
+export interface MetaFileData {
+	phpVersion: string;
+	wordpressVersion: string;
+}
+
+export interface ImporterResult extends Omit< BackupContents, 'metaFile' > {
+	meta?: MetaFileData;
+}
+
+export interface Importer {
+	import( rootPath: string ): Promise< ImporterResult >;
+}
+
+export class DefaultImporter implements Importer {
+	constructor( protected backup: BackupContents ) {}
+
+	async import( rootPath: string ): Promise< ImporterResult > {
+		await this.importDatabase();
+		await this.importWpContent( rootPath );
+		let meta: MetaFileData | undefined;
+		if ( this.backup.metaFile ) {
+			meta = await this.parseMetaFile();
+		}
+		return {
+			extractionDirectory: this.backup.extractionDirectory,
+			sqlFiles: this.backup.sqlFiles,
+			wpContent: this.backup.wpContent,
+			meta,
+		};
+	}
+
+	protected async importDatabase(): Promise< void > {
+		// empty in DefaultImporter
```

Comment:
Each importer should import SQL files if they are present. As we will separately implement that part, should we adjust this comment to make it clearer that we will implement it under another issue?